test/src/test/java/org/apache/accumulo/test/iterator/SummingCombinerTest.java

```diff
@@ -38,21 +38,21 @@
 import org.apache.accumulo.iteratortest.IteratorTestInput;
 import org.apache.accumulo.iteratortest.IteratorTestOutput;
 import org.apache.accumulo.iteratortest.environments.SimpleIteratorEnvironment;
-import org.apache.accumulo.iteratortest.junit4.BaseJUnit4IteratorTest;
+import org.apache.accumulo.iteratortest.junit5.BaseJUnit5IteratorTest;
 import org.apache.accumulo.iteratortest.testcases.IteratorTestCase;
-import org.junit.runners.Parameterized.Parameters;
+import org.junit.jupiter.params.provider.Arguments;
 
 /**
  * Iterator test harness tests for SummingCombiner
  */
-public class SummingCombinerTest extends BaseJUnit4IteratorTest {
+public class SummingCombinerTest extends BaseJUnit5IteratorTest {
 
-  @Parameters
-  public static Object[][] parameters() {
+  @Override
+  protected List<Arguments> parameters() {
     IteratorTestInput input = getIteratorInput();
     IteratorTestOutput output = getIteratorOutput();
     List<IteratorTestCase> tests = IteratorTestCaseFinder.findAllTestCases();
-    return BaseJUnit4IteratorTest.createParameters(input, output, tests);
+    return BaseJUnit5IteratorTest.createParameters(input, output, tests);
   }
```

Comment:
There are a few outliers, but most of them are the exact same. Did you have anything specific in mind for how this could be cleaned up? 

One thing I can think of is remove the `parameters()` method and instead have the extending classes override `getIteratorInput()` `getIteratorOutput()` and maybe `getTests()` moving this duplicate logic into `BaseJUnit5IteratorTest`.

Reply:
Not sure if its much cleaner but I implemented the idea above in [0bb2170](https://github.com/apache/accumulo/pull/2584/commits/0bb2170fbe3d71ba1227136d4689ec0151b61ac2)